lib/widgets/payment_request_dialog.dart

```diff
@@ -127,7 +96,45 @@ class PaymentRequestDialogState extends State<PaymentRequestDialog>
         .listen((settings) => _accountSettings = settings);
 
     _sentPaymentResultSubscription = widget.accountBloc.fulfilledPayments.listen((fulfilledPayment) {
-      // Trigger the collapse animation and show flushbar 200ms after the animation is completed
+      controller = AnimationController(
+          vsync: this, duration: Duration(milliseconds: 600));
+      borderAnimation = Tween<double>(begin: 0.0, end: 8.0).animate(
+          CurvedAnimation(parent: controller, curve: Curves.slowMiddle));
+      opacityAnimation = Tween<double>(begin: 0.0, end: 1.0).animate(
+          CurvedAnimation(parent: controller, curve: Curves.slowMiddle));
+      colorAnimation = new ColorTween(
+        begin: theme.BreezColors.blue[500],
+        end: theme.BreezColors.white[500],
+      ).animate(controller)
+        ..addListener(() {
+          setState(() {});
+        });
+
+      RenderBox _paymentTableBox = widget.tableKey.currentContext
+          .findRenderObject();
+      var _paymentItemStartPosition = _paymentTableBox
+          .localToGlobal(Offset.zero)
+          .dy - 32.0;
```

Comment:
What does the 32.0 stand for?

Reply:
I will test this with another widget to be sure but the start position was at the center of the widget.
One of the methods may calculate distance from safearea's 0.0 while the other uses top of screen. The difference may be the system status bar. If so, I'll use the constant for it.

Reply:
It was the height of status bar. There was a 4.0 dp difference for my screen.